schema/common.mod

```diff
@@ -802,12 +802,12 @@
 -->
 <!ELEMENT dynamics ((p | pp | ppp | pppp | ppppp | pppppp |
 	f | ff | fff | ffff | fffff | ffffff | mp | mf | sf |
-	sfp | sfpp | fp | rf | rfz | sfz | sffz | fz | 
+	sfp | sfpp | fp | rf | rfz | sfz | sffz | fz | n | pf | sfzp |

```

Comment:
One formatting nit - in the DTD files, the style has been to keep the line width to 65 characters for easier reading and reformatting in different uses. Could at least one of these dynamics please move down to the next line with other-dynamics?